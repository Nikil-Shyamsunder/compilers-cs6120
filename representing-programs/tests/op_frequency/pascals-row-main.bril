# Adapted From Patrick LaFontaine's benchmark: https://github.com/sampsyo/bril/blob/main/benchmarks/core/pascals-row.bril
@main {
  v0: int = const 5;
  x: int = id v0;
  v1: int = id x;
  call @generateNthRow v1;
  v2: int = const 0;
}