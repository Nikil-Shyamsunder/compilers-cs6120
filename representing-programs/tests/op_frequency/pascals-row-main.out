call: 1
const: 2
id: 2
